components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -343,7 +343,7 @@ protected void findPreviousAuthenticatedSession(HttpServletRequest request,
                     }
 
                     context.setPreviousSessionFound(true);
-                    sequenceConfig = previousAuthenticatedSeq;
+                    sequenceConfig = previousAuthenticatedSeq.cloneObject();
```

Comment:
@lahirue can we rather use the Object#clone method instead which is the best practice to clone an object?

https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#clone()